.buildkite/macos.integration.pipeline.yml

```diff
@@ -0,0 +1,94 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+
+common:
+  - vault_ec_key_prod: &vault_ec_key_prod
+      elastic/vault-secrets#v0.1.0:
+        path: "kv/ci-shared/platform-ingest/platform-ingest-ec-prod"
+        field: "apiKey"
+        env_var: "EC_API_KEY"
+
+steps:
+  - label: "Packaging: darwin/arm64"
+    key: packaging-macos-arm64
+    env:
+      PLATFORMS: "darwin/arm64"
+      PACKAGES: "tar.gz"
+    command: ".buildkite/scripts/steps/integration-package.sh"
+    artifact_paths:
+      - build/distributions/**
+    retry:
+      automatic:
+        limit: 1
+    agents:
+      provider: "aws"
+      instanceType: "c6g.4xlarge"
+      imagePrefix: "core-ubuntu-2204-aarch64"
+
+  - label: Start ESS stack for macOS
+    key: integration-macos-ess
+    depends_on: packaging-macos-arm64
+    env:
+      ASDF_TERRAFORM_VERSION: 1.9.2
```

Comment:
What's the reason for using this version instead of the defaulted one ?

Reply:
There's PR to the images repo 1483 that explains it in detail. TL;DR: The container image we use here installs terraform:latest. For this version, it's 1.9.2.